### DISCUSSION TITLE
2FA SMS comes 10-15 minutes after requested

### DISCUSSION THREAD

**Discussion by dsimov** [community/community#69680] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

HI.
I tried to sign in today. I tried twice to receive an SMS. I did not. I then signed in with a recovery code, Several minutes later, I receive two SMS authentication codes.

**Comment by santoshvandari** [comment]


**Reply by queenofcorgis** [reply]
@santoshvandari 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment]
Hey there would you mind sharing if you're still experiencing this, what your carrier is, and what country you're located in?
